topics/assembly/tutorials/vgp_genome_assembly/tutorial.md

```diff
@@ -324,6 +324,10 @@ Meryl will allow us to generate the *k*-mer profile by decomposing the sequencin
 >
 {: .comment}
 
+In order to do genome profile analysis, first we need the *k*-mer spectrum of the raw reads, which should hopefully contain information about the genome that you sequenced. These *k*-mers are used to build a histogram (the *k*-mer spectrum), and then the GenomeScope model that fits that data will help infer genome characteristics. To count *k*-mers, we first count them on the separate FASTA files, before merging the counts and generating a histogram based on that. This is a way of parallelizing our work. 
```

Comment:
Suggestion: In order to identify the key metrics of the genome (profile), we start by generating a histogram of the *k*-mer distribution in the raw reads (the *k*-mer spectrum). Then, GenomeScope creates a model fitting the spectrum that allows to estimate the genome characteristics. We work in parallel on each set of raw reads, creating a database of each *k*-mer counts, then merge all the databases by adding the counts of the *k*-mers to build the histogram.

Reply:
incorporated this wording with some edits, does this sound ok? let me know if not !!